multimedia/imagemagick/Makefile

```diff
@@ -0,0 +1,142 @@
+#
+# Copyright (C) 2010-2019 OpenWrt.org
+#
```

Comment:
Remove copyright assignment to OpenWrt.

Reply:
Done. Is this now the recommended practice to avoid the OpenWrt copyright notice?